sharepa/analysis.py

```diff
@@ -10,17 +9,50 @@ def bucket_to_dataframe(name, bucket):
         :type bucket: list[dict]
         :returns: pandas.DataFrame
     '''
-    return pd.DataFrame(
-        {item['key']: item['doc_count'] for item in bucket},
-        index=[name]
-    ).T
+    list_of_dicts = []
+    for item in buckets:
+        single_dict = item.to_dict()
+        single_dict[name] = single_dict.pop('doc_count')
+        if append_name:
+            for key in single_dict.keys():
+                single_dict[append_name + '.' + key] = single_dict.pop(key)
+        list_of_dicts.append(single_dict)
+    return pd.DataFrame(list_of_dicts)
+
+def agg_to_two_dim_dataframe(agg):
+    '''A function that takes an elasticsearch response with aggregation and returns the names of all bucket value pairs
+
+        :param agg: an aggregation from elasticsearch results
+        :type agg: elasticsearch response.aggregation.agg_name object
+        :returns: pandas data frame of one or two dimetions depending on input data
+    '''
+    print agg
```

Comment:
In the future, use python's logging module. If you do use prints, use ```print('some text')``` for python 3 compatibility, and this looks like a debug print so you should remove it.